Examples/Algorithms/TrackFinding/src/TrackFindingAlgorithm.cpp

```diff
@@ -560,9 +560,20 @@ ProcessCode TrackFindingAlgorithm::execute(const AlgorithmContext& ctx) const {
                   (*std::next(secondTrackCopy.trackStatesReversed().begin()))
                       .index();
 
-              Acts::calculateTrackQuantities(trackCandidate);
+              // finalize the track candidate
+
+              if (!m_cfg.reverseSearch) {
+                // these parameters are already extrapolated by the CKF and have
+                // the optimal resolution. note that we did not smooth all the
+                // states.
```

Comment:
Just a question from my ignorance: does it matter that not all states a smoothed? I guess we normally only look at the final resolution, but will we ever need the intermediate states?

Reply:
I guess that is a decision the experiment has to take at the end of the day. For now I only partially smoothed the track because we usually only look at the impact parameters. But if someone wants to study the track state resolutions and pulls we have to refine that a bit